src/observable/dom/AjaxObservable.ts

```diff
@@ -245,6 +245,8 @@ export class AjaxSubscriber<T> extends Subscriber<Event> {
   private serializeBody(body: any, contentType: string) {
     if (!body || typeof body === 'string') {
       return body;
+    } else if (typeof FormData !== 'undefined' && body instanceof FormData) {
```

Comment:
this should check if `root.FormData` rather than just `FormData`. Also, the `typeof` isn't really necessary, a simple `FormData && body instanceof FormData` will work.